### DISCUSSION TITLE
When I damage a monster that is poisoned, why do I get a popup that asks if I want to x2 the damage?

### DISCUSSION THREAD

**Discussion by alecgerona** [Lurkars/gloomhavensecretariat#587] [Q&A] [answered]
If I click check it does double it. As far as I'm aware, it should only add +1 to the damage right?

**Comment by Lurkars** [comment, marked as answer]
I cannot reproduce. Especially on Poison, there a checkmark and a 2x icon, checkmark just adds 1 more damage and 2x (obviously) doubles the damage. Did you accidentally clicked the 2x button? Please test this again, and if you find a good way to reproduce, then document here. By now, everything is working as expected there for me.

**Reply by alecgerona** [reply]
Ah silly me I get it now. I thought clicking the checkmark is what triggers the 2x damage. Didn't notice that the 2x icon itself is a button. Can this apply for shield as well? A popup that can reduce damage as per shield amount?

**Reply by Lurkars** [reply]
Yes, you just need to enable it in settings: Under **Confirm applicable conditions** , click on the shield icon (black condition with a shield an X), that it is not crossed out anymore.

This does also work for character shield, if you enable **Shield and Retaliate** under **Characters** in settings.

**Reply by alecgerona** [reply]
Thanks man love your work!
